brambling/admin/forms.py

```diff
@@ -42,7 +42,7 @@ class PersonChangeForm(forms.ModelForm):
 
     class Meta:
         model = Person
-        fields = ('email', 'password', 'is_active',)
+        fields = ('first_name','last_name','email', 'password', 'is_active',)
```

Comment:
Kk. Once this is reverted I'll merge the PR. It's possible that there are instances of given_name and surname that we've missed, but this seems pretty comprehensive (and it's not easy to find things we might've missed without merging these changes first.)